_src-js/lib/bound-properties.js

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+import ReactDOM from 'react-dom'
+import videojs from 'video.js';
+import shortid from 'shortid';
+
+const Form = React.createClass({
+  getInitialState() {
+    return {
```

Comment:
ah, cool. Didn't really look in other parts of the docs beyond the tutorial.
I might update it while waiting for the contrib-hls issue to resolve.